src/Layout.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+import templates from './Layout.soy.js';
+import Component from 'metal-component';
+import Soy from 'metal-soy';
+
+class Layout extends Component {
```

Comment:
Can you add a simple doc here just explaining generally what this component does? Check out this [example](https://github.com/metal/metal-progressbar/blob/master/src/ProgressBar.js).